languages/en.ini

```diff
@@ -826,6 +826,23 @@ Number = "Number"
 number_decimal_point = "."
 number_thousands_separator = ","
 OAI Server = "OAI Server"
+oauth2_allow_access = "Allow Access"
```

Comment:
I wonder if we should use a more generic prefix than oauth2 -- I assume that this text would be the same for any sort of external permission system, whether the technology is oauth2 or something else. Might be more future-proof to not tie the translation strings to a specific protocol version. Maybe "external_auth_..."?

Reply:
Makes sense, I'll revise. :)